sdk/batch/Microsoft.Azure.Batch/tests/UnitTests/BatchRequestUnitTests.cs

```diff
@@ -249,7 +249,9 @@ public void TestBatchExceptionCreatedWithRetryAfterDateTime()
             batchErrorException.Response.Headers.Add(InternalConstants.RetryAfterHeader, new List<string> { retryAfterString });
             BatchException batchException = new BatchException(batchErrorException);
 
-            Assert.True(TimeSpan.FromSeconds(9) <= batchException.RequestInformation.RetryAfter && batchException.RequestInformation.RetryAfter <= TimeSpan.FromSeconds(10));
+            this.testOutputHelper.WriteLine($"RetryAfter: {batchException.RequestInformation.RetryAfter}");
+            // Give some wiggle room in case tests are running slow
+            Assert.True(TimeSpan.FromSeconds(5) <= batchException.RequestInformation.RetryAfter && batchException.RequestInformation.RetryAfter <= TimeSpan.FromSeconds(10));
```

Comment:
@mattchr do you think this is window will always pass? Would it make more since to just remove the lower bound? Even with that do we every expect that it would be higher then 10 in any cases?

Reply:
I removed the lower bound.

It shouldn't really ever be higher than 10 since I've set it to 10 above - unless the clock goes backwards (which can probably happen... leap seconds?) - but I've gotta imagine that's really rare. If we see issues with it still after this we can skip it until I can fix it better.